src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String COLOUR = "white";
+    private static final int RADIUS = 10;
+    private static final int MAXSIZE = 13;
+    private static final int TOTAL_FIGURE_AMOUNT = 6;

```

Comment:
why 6? you have only 5 figures

Reply:
it was in task..
"In the main() method we need to create an array of figures (the size of array can be 3 or 6, it doesn't matter). "

Reply:
it means how many figures you created it will be size of your array. you created 5 figures